### PR TITLE
Buster arm patch 3: the unscammening 

### PR DESCRIPTION
**Pull request by LazennG**
# Document the changes in your pull request
image

Watching buster arm and getting feedback showed me where the buster arm consistently falls flat on its face and also where it should be changed, along with fixing a few bugs i missed.
Bugfixes:
-Wire snatch no longer blows you the fuck up on hitting a fuel tank
-left arm and hand sprite recentered so they no longer poke out of clothes more than they should 
-fixed a bug allowing grapple and megabuster to affect people in front of the user regardless of the attack landing its first hit
-animations made to break much much less
Other changes:
-new afterimage effect for new status effect
-buster arm's description in uplink now mentions how conspicuous the arm is and how gloves can't be worn
-buster arm's hp lowered to 50 and flat resistances to burn and brute have been removed so disabling it in a fight with attacks and emps becomes an available option

New status effect called double down:
When the user doesn't have the element of surprise or is fighting someone who noticed the arm then the fight is very much disadvantaged against the user, especially due to the noisy nature of the moves that will often attract attention. To help remedy this, a new status effect is introduced to the arm's moveset. On hitting a valid target for any of the moves, like a vending machine or person for grapple, a mob for mop the floor, or something solid for wire snatch, the user will get the buff reducing incoming damage by 65% for 2 seconds along for letting them ignore damage slowdown that can be extended should the user keep hitting valid targets, encouraging them to be aggressive. 

dreamseeker_e7Z9RqlPEX

The 65% may be cause for awooga and similar reactions but testing with multiple people showed that it was an appropriate number considering how during fights users usually take at least 2 disabler shots just approaching people who notice them and that this damage reduction still lets the arm have function in the common case of multiple hostiles in the room. Additionally, The slowdown ignoring is still counterable by bolas along with the entire moveset being disabled for like 13 seconds with an emp and even longer if the arm gets fucked up enough.

# Spriting

bingus

# Wiki Documentation

Probably something about how landing moves will momentarily grant the user a 65% boost to defenses when landing moves.

# Changelog

:cl:  @LazennG @thegoldencat413 @slicerv @solwashere @Burger1998 @DeadlyBag @milk-is-cool 
rscadd: Added damage reducing status effect for buster arm and longer lasting afterimage effect
bugfix: fixed buster arm animations and wire snatch function
tweak: tweaked buster arm effectiveness and durability
imageadd: added new icon for new status
/:cl:
thanks to goldencat for the sprite & idea and everyone on the list for feedback and testing

### CODE REVIEW

**Comment by ToasterBiome** [comment]
ooooo

**Review by ynot01 — DISMISSED** [review]
cool changelog

**Comment by ynot01** [comment]
mid PR

**Comment by LazennG** [comment]
your mother

**Comment by 00ze-cyclone** [comment]
The best defense once again prove to be offense

**Comment by MajManatee** [comment]
if only we had keybindable action buttons...

**Review by cuackles — APPROVED** [review]
icon do be looking good, visual effect also good

**Comment by ynot01** [comment]
i am happy that the resistance status is very telling, not enough clear visual tells these days

File: code/datums/status_effects/buffs.dm [hunk 1]
```diff
@@ -588,4 +588,47 @@
...
+	if(ishuman(owner))
+		qdel(F)
+		var/mob/living/carbon/human/H = owner
+		H.unignore_slowdown(type)
+		H.physiology.brute_mod *= 2.85
+		H.physiology.burn_mod *= 2.85
+		H.physiology.tox_mod *= 2.85
+		H.physiology.oxy_mod *= 2.85
+		H.physiology.clone_mod *= 2.85
+		H.physiology.stamina_mod *= 2.85
```
Comment (ynot01) [inline code/datums/status_effects/buffs.dm]: Please use `/= 0.35` instead to be uniform with the above

File: code/datums/status_effects/buffs.dm [hunk 2]
```diff
@@ -588,4 +588,47 @@
...
-	icon_state = "time_dilation" //yogs end
+	icon_state = "time_dilation" 
+
+/datum/status_effect/doubledown
+	id = "doubledown"
+	duration = 20
+	tick_interval = 0
+	status_type = STATUS_EFFECT_REFRESH
+	alert_type = /atom/movable/screen/alert/status_effect/doubledown
+	var/obj/effect/temp_visual/decoy/tensecond/F //surely a combo wont go on for more than 10 seconds
```
Comment (ynot01) [inline code/datums/status_effects/buffs.dm]: Awful variable name

File: code/game/objects/items/devices/busterarm/_buster.dm [hunk 3]
```diff
@@ -47,3 +47,13 @@
...
 		target.apply_damage(damage, BRUTE, limb_to_hit, armor, wound_bonus=CANT_WOUND)
+
+//knocking them down
+/datum/action/cooldown/buster/proc/footsies(mob/living/target)
+	if(target.stat == CONSCIOUS && target.resting == FALSE && !target.has_status_effect(STATUS_EFFECT_PARALYZED) && !target.has_status_effect(STATUS_EFFECT_KNOCKDOWN))
+		animate(target, transform = matrix(90, MATRIX_ROTATE), time = 0.1 SECONDS, loop = 0)
+
+//Check for if someone is allowed to be stood back up
+/datum/action/cooldown/buster/proc/wakeup(mob/living/target)
+	if(target.stat == CONSCIOUS && target.resting == FALSE && !target.has_status_effect(STATUS_EFFECT_PARALYZED) && !target.has_status_effect(STATUS_EFFECT_KNOCKDOWN))
```
Comment (ynot01) [inline code/game/objects/items/devices/busterarm/_buster.dm]: Is there a reason you don't just use `MOBILITY_STAND` to knock people over and detect if they're on the floor?

**Review by ynot01 — DISMISSED** [review]

File: code/game/objects/items/devices/busterarm/_buster.dm (hunk 3 above)
Comment (LazennG) [reply code/game/objects/items/devices/busterarm/_buster.dm]: hadnt heard of it before today

Comment (LazennG) [reply code/game/objects/items/devices/busterarm/_buster.dm]: need to hold off on the pr because this is acting up a bit

**Comment by LazennG** [comment]
dreamseeker_AS9TEDSPty
mop animation working as intended now

**Pull request merged**